apps/web/src/components/Settings/Dispatcher/ToggleDispatcher.tsx

```diff
@@ -22,6 +24,7 @@ interface Props {
 }
 
 const ToggleDispatcher: FC<Props> = ({ buttonSize = 'md' }) => {
+  const { data: idKitData } = useIsIdKitVerifiedQuery({});
```

Comment:
```suggestion
  const { data: idKitData } = useIsIdKitVerifiedQuery();
```

do we need to pass empty object?